test/.env.dev

```diff
@@ -0,0 +1,8 @@
+CONTENT_SERVER_URL=http://127.0.0.1:3030
+SMTP_HOST=127.0.0.1
+SMTP_PORT=9999
+SMTP_SECURE=false
+LOG_LEVEL=info
+CUSTOMS_SERVER_URL=none
+LOCKOUT_ENABLED=true
+SNS_TOPIC_ARN=disabled
```

Comment:
Should this file be `.env.test`? I'm assuming NODE_ENV is set to `TEST` when the tests are running on Travis, though I guess it's set to DEV when running locally. :) Just thought I'd point this out if it makes sense to rename it.